hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -39,6 +39,15 @@ Block 1 "3318159292" "data_o = 39'(data_i);"
 INSTANCE: tb.dut.u_tl_adapter_rom.u_tlul_data_integ_enc_instr.u_data_gen
 Block 1 "3318159292" "data_o = 39'(data_i);"
 
+INSTANCE: tb.dut.gen_rom_scramble_enabled.u_rom.u_rom.u_prim_rom.gen_generic.u_impl_generic
+ANNOTATION: "cfg_i is tied to 0 in tb"
```

Comment:
I think this needs to be `rom_cfg_i` (which is the name it has as a dut port). I'd probably also:
- Say something like "We expect this port to be wired to a constant."
- Add an assertion in `rom_ctrl.sv` that says this is constant. Maybe something like `ASSERT(CfgStable_A, $stable(rom_cfg_i))` ?